circle.yml

```diff
@@ -62,23 +62,27 @@ jobs:
     steps:
       - checkout
       - run:
-          name: Generate Maven credentials
+          name: Generate Bintray credentials
           shell: /bin/bash -euo pipefail
           command: |
-            aws s3 cp s3://mapbox/android/signing-credentials/secring.gpg secring.gpg
-            echo "NEXUS_USERNAME=$PUBLISH_NEXUS_USERNAME
-            NEXUS_PASSWORD=$PUBLISH_NEXUS_PASSWORD
-            signing.keyId=$SIGNING_KEYID
-            signing.password=$SIGNING_PASSWORD
-            signing.secretKeyRingFile=../secring.gpg" >> gradle.properties
+            if [ -n "${BINTRAY_USER}" ]; then
+              echo "BINTRAY_USER=$BINTRAY_USER
+              BINTRAY_API_KEY=$BINTRAY_API_KEY
+              GPG_PASSPHRASE=$GPG_PASSPHRASE"
+            fi
       - run:
           name: Build libraries
           command: make release
       - deploy:
-          name: Publish to Maven Central
+          name: Publish to Bintray
           command: |
-            if [ "${CIRCLE_BRANCH}" == "master" ]; then
-            make publish ;
+            if [[ $CIRCLE_BRANCH == master ]] || [[ $CIRCLE_BRANCH == release-v* ]] || [[ $CIRCLE_TAG == v* ]]; then
```

Comment:
We are using the `v[number]` convention in this repo, so `[[ $CIRCLE_BRANCH == release-v* ]]` can be skipped.

Reply:
Removed